src/main/java/com/redhat/devtools/lsp4ij/console/explorer/actions/ExportServerAction.java

```diff
@@ -0,0 +1,128 @@
+        if (this.languageServerDefinitions.size() == 1) {
+            getTemplatePresentation().setText(LanguageServerBundle.message("action.lsp.console.explorer.export.server.text"));
+            getTemplatePresentation().setDescription(LanguageServerBundle.message("action.lsp.console.explorer.export.server.description"));
+        } else {
+            getTemplatePresentation().setText(LanguageServerBundle.message("action.lsp.console.explorer.export.servers.text"));
+            getTemplatePresentation().setDescription(LanguageServerBundle.message("action.lsp.console.explorer.export.servers.description"));
+        }
+        getTemplatePresentation().setIcon(AllIcons.ToolbarDecorator.Export);
+    }
+
+    @Override
+    public void actionPerformed(@NotNull AnActionEvent e) {
+        FileChooserFactory fileChooserFactory = FileChooserFactory.getInstance();
+        FileSaverDescriptor fileSaverDescriptor = new FileSaverDescriptor(
+                LanguageServerBundle.message("action.lsp.console.explorer.export.servers.zip.save.title"), LanguageServerBundle.message("action.lsp.console.explorer.export.servers.zip.save.description"));
+        FileSaverDialog fileSaverDialog = fileChooserFactory.createSaveFileDialog(fileSaverDescriptor, e.getProject());
+        VirtualFileWrapper fileWrapper = fileSaverDialog.save("export.zip");
+        if (fileWrapper != null) {
+            VirtualFile virtualFile = fileWrapper.getVirtualFile(true);
+            if (virtualFile != null) {
+                ApplicationManager.getApplication().runWriteAction(() -> {
+                    try {
+                        virtualFile.setBinaryContent(createZipFromLanguageServers());
+                    } catch (IOException ex) {
+                        LOGGER.warn(ex.getLocalizedMessage(), e);
+                    }
+                });
+            }
+        }
+    }
+
+    /**
+     * Creates a zip file by handling each user defined language server definitions
+     * @return zip file as a byte array
+     */
+    private byte[] createZipFromLanguageServers() throws IOException {
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        ZipOutputStream zos = new ZipOutputStream(baos);
+
+        for (LanguageServerDefinition lsDefinition : languageServerDefinitions) {
+            Gson gson = new GsonBuilder()
+                    .registerTypeAdapter(UserDefinedLanguageServerDefinition.class, new LanguageServerDefinitionSerializer())
+                    .setPrettyPrinting()
+                    .create();
+            String json = gson.toJson(lsDefinition);
+            String initializationOptions = ((UserDefinedLanguageServerDefinition) lsDefinition).getInitializationOptionsContent();
+            String settings = ((UserDefinedLanguageServerDefinition) lsDefinition).getConfigurationContent();
+            String lsName = lsDefinition.getDisplayName();
+
+            writeToZip(lsName + "/template.json", json, zos);
```

Comment:
Please define `template.json` file name in a constant and reuse this constant and reuse this constant in your `loadFromTemplate` method.

Do that for other file name.